Sergey_kravchuk/02.Git.Local/README.md

```diff
@@ -0,0 +1,21 @@
+# 265  cd c/Users/s.kravchuk/
```

Comment:
This history is history of your commands how prepare home work, but not home work itself

I need list commands, how you did this: https://github.com/pluhin/sa.it-academy.by/wiki/02.-GIT.-Local